### PR TITLE
Dev raghu

### PR DESCRIPTION
**Pull request by rjraghutech**

### CODE REVIEW

File: src/Components/Header/Header.js [hunk 1]
```diff
@@ -1,15 +1,39 @@
-import React from 'react'
-
-import '../Header/header.css'
+import React, { useState } from "react";
 
+import "../Header/header.css";
```
Comment (Souvikdeb2612) [inline src/Components/Header/Header.js]: format imports

File: src/Components/Header/Login.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+  return (
+    <>
+      <div className="form2">
+        {!user ?
+          <form className="form">
+            <input type="text" name="username" placeholder="UserName" />
+            <input type="password" placeholder="Password" name="password"/>
+            <button type="submit" className="loginBtn">Login</button>
+            <button onClick={() => setUser(true)} className="createBtn">Don't have account..? Sign In</button>
```
Comment (Souvikdeb2612) [inline src/Components/Header/Login.js]: create a separate function for this

File: src/Components/Header/Login.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+          </form>
+          :
+          <div className="form1">
+            <input type="text" name="username" placeholder="UserName" /><br />
+            <input type="text" name="emailId" placeholder="Email ID"></input><br></br>
+            <input type="password" placeholder="Enter Password" name="password"/><br />
+            <input type="password" placeholder="Re-enter Password" name="password"/><br></br>
+            <button type="submit" className="signupBtn">Sign UP</button><br></br>
+            <button onClick={() => setUser(false)} className="createBtn">Already have account Login...</button>
```
Comment (Souvikdeb2612) [inline src/Components/Header/Login.js:30]: same as above

**Review by Souvikdeb2612 — CHANGES_REQUESTED** [review]

File: src/Components/Header/Login.js (hunk 3 above)
Comment (rjraghutech) [reply src/Components/Header/Login.js:30]: I have updated the requested changes.

File: src/Components/Header/Header.js (hunk 1 above)
Comment (rjraghutech) [reply src/Components/Header/Header.js]: I have updated the things sir.

File: src/Components/Header/Login.js (hunk 2 above)
Comment (rjraghutech) [reply src/Components/Header/Login.js]: I have updated the changes sir.

File: src/Components/Header/Login.js [hunk 4]
```diff
@@ -11,7 +12,7 @@ export function Login() {
     <form className="form">
     <input type="text" name="username" placeholder="UserName" />
     <input type="password" placeholder="Password" name="password"/>
-    <button type="submit" className="loginBtn">Login</button>
+    <NavLink to={"/App"}><button type="submit" className="loginBtn">Login</button></NavLink>
     <button onClick={() => setUser(true)} className="createBtn">Don't have account..? Sign In</button>
```
Comment (Souvikdeb2612) [inline src/Components/Header/Login.js]: create separate function

File: src/Components/SideBar/SideBar.js [hunk 5]
```diff
@@ -1,43 +1,46 @@
...
           <button type="button" className="btnside">
-            <img src={logo} alt="logo" srcset="" />
-            <span className="brand">STOKIDA</span>
+            {/* <NavLink to={"/App"}> */}
+              <img src={logo} alt="logo" srcset="" />
+              <span className="brand">STOKIDA</span>
+            {/* </NavLink> */}
           </button>
-          <button type="button" className="btnside">
+          <button type="button" className="btnside sidebtn">
```
Comment (Souvikdeb2612) [inline src/Components/SideBar/SideBar.js:19]: These are navlinks not buttons

**Review by Souvikdeb2612 — CHANGES_REQUESTED** [review]

**Review by Souvikdeb2612 — APPROVED** [review]

**Pull request merged**